docs/source/api/v4/servers_id.rst

```diff
@@ -326,6 +327,7 @@ Allow user to delete server through api.
 
 :Auth. Required: Yes
 :Roles Required: "admin" or "operations"
+:Permissions Required: SERVER:DELETE, SERVER:READ, CACHE-GROUP:UPDATE, DELIVERY-SERVICE:UPDATE, CDN:UPDATE, SERVER:READ, DELIVERY-SERVICE:READ, CDN:READ, PHYSICAL-LOCATION:READ, CACHE-GROUP:READ, TYPE:READ, PROFILE:READ
```

Comment:
Same as above RE: `DELIVERY-SERVICE:UPDATE`, `CACHE-GROUP:UPDATE`, and `CDN:UPDATE`.